tools/rv_histogram.c

```diff
@@ -189,39 +194,38 @@ void print_hist_stats(const rv_hist_t *stats, size_t stats_size)
     }
 }
 
-void reg_hist_incr(rv_insn_t *ir)
+static void reg_hist_incr(const rv_insn_t *ir)
 {
-    if (!ir)
-        return;
+    if (ir != NULL) {
```

Comment:
Don't change the function body.

Reply:
Check https://github.com/sysprog21/rv32emu/blob/master/CONTRIBUTING.md carefully: Avoid unnecessary nesting levels.

Reply:
Modifications similar to the single-line if statement have been restored.